analysis/R/association.R

```diff
@@ -353,10 +352,14 @@ ComputeDistributionEM <- function(reports, report_cohorts, maps,
     f <- var_params$f
     pstar <- (1 - f / 2) * p + (f / 2) * q
     qstar <- (1 - f / 2) * q + (f / 2) * p
+    k <- var_params$k
 
-    if (ignore_other) {
+    # Ignore other probability if either ignore_other is set or k == 1
```

Comment:
ignore_other is set globally across both (or many) RAPPOR variables. We have to use ignore_other code for strings, but ignore it for booleans.